curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/6353004b235d7a2e0b913f2b.md

```diff
@@ -43,7 +43,7 @@ assert.match(getStandardDeviation.toString(), /standardDeviation\s*=\s*Math\.pow
 Your `Math.pow()` function should have a base of `variance` and an exponent of `1/2`.
 
 ```js
-assert.match(getStandardDeviation.toString(), /standardDeviation\s*=\s*Math\.pow\(\s*variance\s*,\s*1\s*\/\s*2\s*\)/);
+assert.match(getStandardDeviation.toString(), /standardDeviation\s*=\s*Math\.pow\(\s*variance\s*,\s*((1\s*\/\s*2)|0.5)\s*\)/);
```

Comment:
```suggestion
assert.match(getStandardDeviation.toString(), /standardDeviation\s*=\s*Math\.pow\(\s*variance\s*,\s*(\(\s*1\s*\/\s*2\s*\)|1\s*\/\s*2|0\.5|\.5)\s*\);?\s*\}/);
```
Couple more variations:
- `(1/2)`
- `1/2`
- `0.5`
- `.5`

I've thrown at the end `\}` to not match when parenthesis is messed up, ie.:
```js
const getStandardDeviation = (array) => {
  const variance = getVariance(array);
  standardDeviation = Math.pow(variance, .5)) 
}
```

Reply:
@gikf thanks